libs/src/sdk/api/chats/serverTypes.ts

```diff
@@ -0,0 +1,148 @@
+// NOTE: No imports allowed - quicktype is not yet able to track imports!
+
+export type ChatCreateRPC = {
+  method: 'chat.create'
+  params: {
+    chat_id: string
+    invites: Array<{
+      user_id: string
+      invite_code: string
+    }>
+  }
+}
+
+export type ChatDeleteRPC = {
+  method: 'chat.delete'
+  params: {
+    chat_id: string
+  }
+}
+
+export type ChatInviteRPC = {
+  method: 'chat.invite'
+  params: {
+    chat_id: string
+    invites: Array<{
+      user_id: string
+      invite_code: string
+    }>
+  }
+}
+
+export type ChatMessageRPC = {
+  method: 'chat.message'
+  params: {
+    chat_id: string
+    message_id: string
+    message: string
+    parent_message_id?: string
+  }
+}
+
+export type ChatReactRPC = {
+  method: 'chat.react'
+  params: {
+    chat_id: string
+    message_id: string
+    reaction: string
```

Comment:
`reaction` should be `int` for the enum like in discovery/identity, right?

Reply:
Yeah you're right...

although... we _could_ take a new route and put the actual emojis in there as "😍 🔥 🥳 🤯 " are all strings. I think we were constrained initially by Solana storage size and not wanting to do expensive validation in the program (technically theres no Solana program but it was built with a migration to solana in mind), but with our validator on go-comms I think we could do it...

Reply:
oo interesting. i dont have a strong preference either way; we can do whatever is easiest for you on the client side. though UTF-8 strings sound more easily extensible